doc/man3/OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY.pod

```diff
@@ -0,0 +1,109 @@
+=pod
+
+=head1 NAME
+
+OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY,
+OSSL_DESERIALIZER_CTX_set_cipher,
+OSSL_DESERIALIZER_CTX_set_passphrase,
+OSSL_DESERIALIZER_CTX_set_passphrase_cb,
+OSSL_DESERIALIZER_CTX_set_passphrase_ui
+- Deserializer routines to deserialize EVP_PKEYs
+
+=head1 SYNOPSIS
+
+ #include <openssl/deserializer.h>
+
+ OSSL_DESERIALIZER_CTX *OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY(const EVP_PKEY *pkey,
+                                                              const char *propquery);
+
+ int OSSL_DESERIALIZER_CTX_set_cipher(OSSL_DESERIALIZER_CTX *ctx,
+                                      const char *cipher_name,
+                                      const char *propquery);
+ int OSSL_DESERIALIZER_CTX_set_passphrase(OSSL_DESERIALIZER_CTX *ctx,
+                                          const unsigned char *kstr,
+                                          size_t klen);
+ int OSSL_DESERIALIZER_CTX_set_passphrase_cb(OSSL_DESERIALIZER_CTX *ctx,
+                                             pem_password_cb *cb, void *cbarg);
+ int OSSL_DESERIALIZER_CTX_set_passphrase_ui(OSSL_DESERIALIZER_CTX *ctx,
+                                             const UI_METHOD *ui_method,
+                                             void *ui_data);
+
+=head1 DESCRIPTION
+
+OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY() is a utility function that
+creates a B<OSSL_DESERIALIZER_CTX>, finds all applicable deserializer
+implementations and sets them up, so all the caller has to do next is
+to call functions like OSSL_DESERIALIZE_from_bio().
+
+What OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY() does in more detail is to
```

Comment:
Maybe change to
Internally OSSL_DESERIALIZER_CTX_new_by_EVP_PKEY() searches for all available L<EVP_KEYMGMT(3)> implementations, and then builds a list of all potential deserializers that may be able to process the serialized input into data suitable for B<EVP_PKEY>s.

Reply:
Thanks!  Changed